ecs-parent-pom/pom.xml

```diff
@@ -69,7 +69,7 @@
               <id>docker-image</id>
               <phase>package</phase>
               <goals>
-                <goal>build-nofork</goal>
+                <goal>build</goal>
```

Comment:
The build-nofork was something they had introduced in 0.18.1 with a behavioral change to the build goal. In 0.18.1, the build goal also carried out, compile, package, which was directly contradictory to how we were utilizing it (attaching the build to the package phase). 

In 0.19 or 0.20 they rolled back that change and removed nofork so it behaved as it did before with the build goal only triggering the build of the docker image and not also triggering prior phases.

In short, build-nofork is no more, and build is was again the way to build the docker image.